Cargo.toml

```diff
@@ -19,6 +19,8 @@ license = "MPL-2.0"
 
 [features]
 default = []
+# Make Vulkan backend available on platforms where it is by default not, e.g. macOS
+gfx-backend-vulkan = ["wgn/gfx-backend-vulkan"]
```

Comment:
This is a bit tricky, since the feature is not available on all platforms (only iOS and macOS). Need to figure out how cargo treats this.

Reply:
Isn't the feature available everywhere, just not really used except macOS?

Reply:
Speaking in terms of strictly cargo features, it's only optional on iOS/macOS. On other platforms it's not optional, therefore not a feature.